web/packages/design/src/SlidePanel/SlidePanel.tsx

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
This component can replace the one used in the access request component once merged.